HerPublicWebsite.BusinessLogic/Services/EligiblePostcode/EligiblePostcodeListService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Reflection;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace HerPublicWebsite.BusinessLogic.Services.EligiblePostcode;
+
+public interface IEligiblePostcodeListService
+{
+    public List<string> GetEligiblePostcodes();
+}
```

Comment:
Don't think we actually benefit from having this interface do we? Or expect to meaningfully use it at any point. Would just drop it if it's boilerplate.

Reply:
have removed